lib/src/core/engine.dart

```diff
@@ -823,11 +823,8 @@ class Engine extends Disposable with EventsEmittable<EngineEvent> {
   }) async {
     final previousAnswer =
         (await subscriber?.pc.getLocalDescription())?.toPBType();
-    final previousOffer =
```

Comment:
can you clarify why this was removed?

Reply:
There was an internal exception in flutter-webrtc, so I temporarily removed the code (added from 1.5.5) and added it back after testing
<img width="1349" alt="image" src="https://github.com/livekit/client-sdk-flutter/assets/17733746/0aa9bae4-4994-4b01-b9b7-a5d29f675d83">